src/snackbar/Snackbar.spec.js

```diff
@@ -46,6 +46,26 @@ describe('Snackbar', () => {
     expect(createPortal).toBeCalledWith(<Snackbar {...props} />, document.body);
   });
 
+  it('accepts node as text', async () => {
```

Comment:
Following up on the previous comment, as node can _be_ a string, we could clarify:
```suggestion
  it('accepts element as text', async () => {
```